samples/nrf9160/https_client/src/main.c

```diff
@@ -114,6 +114,13 @@ int tls_setup(int fd)
 		TLS_SEC_TAG,
 	};
 
+#if defined(CONFIG_SAMPLE_TFM_MBEDTLS)
+	err = tls_credential_add(tls_sec_tag[0], TLS_CREDENTIAL_CA_CERTIFICATE, cert, sizeof(cert));
+	if (err) {
+		return err;
+	}
+#endif
```

Comment:
Why not move this next to code that provisions the certificates to the modem?

Reply:
Mainly because I didn't want to introduce the `tls_sec_tag` again or make it global.

Reply:
Sorry, I meant why don't we put this piece of code next to the bit that calls `cert_provision()` a few lines below?
This way we show the two different calls that provision the certificates one next to the other.

Reply:
The call for `cert_provision()` is on the main-function and this call is in the `tls_setup()` -function. The `tls_sec_tag` is introduced in the `tls_setup()`-function as well. I know what you mean, but I wanted to make only minimal changes to the sample.